### ISSUE TITLE
Build failure: wxpython

### ISSUE THREAD

**Issue by 2403772980ygy** [NixOS/nixpkgs#349248]
### Steps To Reproduce

1. build wxpython

### Build log

[wx.txt](https://github.com/user-attachments/files/17410139/wx.txt)

### Notify maintainers

@mweinelt 

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.54, NixOS, 24.11 (Vicuna), 24.11.20241008.f6f24b0`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.8`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/ycfb5fhsyk1h1pr5ql2lhdsnydl4l627-source`
```

---

**Comment by devurandom** [comment]
```
error: builder for '/nix/store/fnsysx7h408wq4nc5ni9jggi5b0y0cpy-python3.11-wxpython-4.2.1.drv' failed with exit code 1;
       last 10 log lines:
       > Waf: Entering directory `/build/wxPython-4.2.1/build/waf/3.11/gtk3'
       > **** Compiler: gcc --version
       >      gcc (GCC) 13.3.0
       >      Copyright (C) 2023 Free Software Foundation, Inc.
       >      This is free software; see the source for copying conditions.  There is NO
       >      warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
       > Waf: Leaving directory `/build/wxPython-4.2.1/build/waf/3.11/gtk3'
       > source not found: 'sip/siplib/bool.cpp' in bld(source=['sip/siplib/apiversions.c', 'sip/siplib/bool.cpp', 'sip/siplib/descriptors.c', 'sip/siplib/int_convertors.c', 'sip/siplib/objmap.c', 'sip/siplib/qtlib.c', 'sip/siplib/sip_array.c', 'sip/siplib/siplib.c', 'sip/siplib/threads.c', 'sip/siplib/voidptr.c'], target='siplib', meths=['check_err_features', 'check_err_order', 'process_rule', 'process_source', 'set_bundle', 'apply_bundle', 'init_pyext', 'apply_link', 'apply_implib', 'process_use', 'propagate_uselib_vars', 'apply_incpaths', 'apply_vnum', 'set_macosx_deployment_target'], features=['c', 'cxx', 'cshlib', 'cxxshlib', 'pyext'], path=/build/wxPython-4.2.1, idx=1, tg_idx_count=1, uselib='siplib WX WXPY', posted=True) in /build/wxPython-4.2.1
       > Command '"/nix/store/0bygl9k94j0xkxq7cmxnpqdqf8va5k7j-python3-3.11.10/bin/python3.11" /build/wxPython-4.2.1/bin/waf --verbose --wx_config=wx-config --gtk3 --python="/nix/store/0bygl9k94j0xkxq7cmxnpqdqf8va5k7j-python3-3.11.10/bin/python3.11" --out=build/waf/3.11/gtk3 configure build ' failed with exit code 1.
       > Finished command: build_py (0m4.350s)
```

**Comment by mweinelt** [comment]
I plan to drop maintainership for wxpython, since I lost any motivation to do so. Sorry!

**Comment by l0b0** [comment]
Can this be closed after #349353?
